src/components/box/box.spec.tsx

```diff
@@ -8,9 +8,19 @@ import {
   testStyledSystemPosition,
   assertStyleMatch,
 } from "../../__spec_helper__/test-utils";
-import Box, { OverflowWrap, ScrollVariant, BoxSizing } from "./box.component";
+import Box, {
+  OverflowWrap,
+  ScrollVariant,
+  BoxSizing,
+  AllowedNumericalValues,
+  Gap,
+} from "./box.component";
 import boxConfig from "./box.config";
 
+const numericalGapValues = Array.from({ length: 9 }).map(
+  (_, i) => i as AllowedNumericalValues
+);
```

Comment:
I'm not going to suggest a change as I'm not sure how else to do this (convert a union type to an array of all its values - note than the other way round is easier so that might be something to consider), but this is potentially dangerous. If we ever add or remove allowed numeric values from the type (not likely I know but who's to know what design changes might happen in future?), this won't have any effect on `numericalGapValues` which will always be the array of integers from 0 to 8.

I think I prefer the alternative of defining an array explicitly and generating the type from that.

Reply:
If we remove/ or add to the type we will need to update the map so the tests would either fail or we would not have full coverage

Reply:
End up updating it after thinking about it a bit more, still think any changes would have been caught but changing the approach to this will allow it to be caught faster